hadoop-ozone/integration-test/src/test/resources/ozone-site.xml

```diff
@@ -26,6 +26,16 @@
     <value>org.apache.hadoop.hdds.fs.MockSpaceUsageCheckFactory$None</value>
   </property>
 
+  <property>
+    <name>ozone.om.transport.class</name>
+    <value>org.apache.hadoop.ozone.om.protocolPB.Hadoop3OmTransportFactory</value>
+  </property>
+
+  <property>
+    <name>ozone.om.s3.grpc.server_enabled</name>
+    <value>false</value>
+  </property>
```

Comment:
I presume this is to avoid breaking some of the integration tests?

If so, which integration tests are broken once we enable this by default? And how well is the new default tested so far?

Reply:
@smengcl thanks for looking at this PR. There are too many integration tests failing with GRPC enabled. I launched a workflow to go over the failures. Please take a look [here](https://github.com/xBis7/ozone/actions/runs/3361441746).

> And how well is the new default tested so far?

The performance numbers have been checked by @michelsumbul.

Reply:
I see **most** are just port binding failures. Possibly a docker compose config issue. If the port number has been changed with the enablement of gRPC the docker image might need to open those new ports as well. Try opening up port 8981 in the corresponding `docker-compose.yaml` files?

```
Error:  org.apache.hadoop.fs.ozone.TestOzoneFsHAURLs  Time elapsed: 7.445 s  <<< ERROR!
java.io.IOException: Failed to bind to address 0.0.0.0/0.0.0.0:8981
	at io.grpc.netty.NettyServer.start(NettyServer.java:328)
	at io.grpc.internal.ServerImpl.start(ServerImpl.java:183)
	at io.grpc.internal.ServerImpl.start(ServerImpl.java:92)
	at org.apache.hadoop.ozone.om.GrpcOzoneManagerServer.start(GrpcOzoneManagerServer.java:123)
```
https://github.com/xBis7/ozone/actions/runs/3361441746/jobs/5571845363

```
Error:  Errors: 
Error:    Failed to bind to address 0.0.0.0/0.0.0.0:8981
Error:    Failed to bind to address 0.0.0.0/0.0.0.0:8981
Error:    Failed to bind to address 0.0.0.0/0.0.0.0:8981
Error:    TestAddRemoveOzoneManager.testBootstrap:173->setupCluster:94 » IO Failed to bi...
Error:    TestAddRemoveOzoneManager.testDecommission:347->setupCluster:94 » IO Failed to...
Error:    TestAddRemoveOzoneManager.testForceBootstrap:283->setupCluster:94 » IO Failed ...
Error:    TestOMRatisSnapshots.init:110 » IO Failed to bind to address 0.0.0.0/0.0.0.0:8...
Error:    TestOMRatisSnapshots.init:110 » IO Failed to bind to address 0.0.0.0/0.0.0.0:8...
...
```
https://github.com/xBis7/ozone/actions/runs/3361441746/jobs/5571845635

Reply:
@smengcl the failing ones are integration tests, no docker involved.  I think fixing them requires using random port in mini cluster for the grpc server in OM.

Reply:
@adoroszlai You are right. Multiple OMs are trying to bind to the same port, causing this issue.

Reply:
Mini cluster tries to configure Grpc server ports on multiple OMs with the same port. The Grpc server is not used by any of the integration tests. We disable it in the `ozone-site` because it's not needed, there are no integration tests for the S3G.

Reply:
Monotonically increasing port numbers, not random, pls. 

#noflakytests